From 2b3cfd4f9a52711f71bf909b36e10084081cf1d5 Mon Sep 17 00:00:00 2001
From: Daniel Diaz <ddiaz@ti.com>
Date: Thu, 2 Sep 2010 15:42:39 -0500
Subject: [PATCH 8/8] camerasrc: Fix building problem.

Fix a couple of building errors:
>  CC     libgstcamerasrc_0.10_la-gstbasecamerasrc.lo
> In file included from ../../../../gst-libs/gst/camerasrc/
>                                                   gstbasecamerasrc.c:35:
> ../../../../gst-libs/gst/camerasrc/gstbasecamerasrc.h:28: fatal error:
> gst/interfaces/photography.h: No such file or directory
> compilation terminated.
> make[4]: *** [libgstcamerasrc_0.10_la-gstbasecamerasrc.lo] Error 1

and:
>  CC     libgstcamerasrc_0.10_la-gstbasecamerasrc.lo
> In file included from ../../../../gst-libs/gst/camerasrc/
>                                                   gstbasecamerasrc.h:28,
>                  from ../../../../gst-libs/gst/camerasrc/
>                                                   gstbasecamerasrc.c:35:
> /opt/gst/include/gstreamer-0.10/gst/interfaces/photography.h:33: fatal
> error: gst/interfaces/photography-enumtypes.h: No such file or directory

Signed-off-by: Daniel Diaz <ddiaz@ti.com>
---
 gst-libs/gst/camerasrc/Makefile.am |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/gst-libs/gst/camerasrc/Makefile.am b/gst-libs/gst/camerasrc/Makefile.am
index 6d5e0fe..5b939f6 100644
--- a/gst-libs/gst/camerasrc/Makefile.am
+++ b/gst-libs/gst/camerasrc/Makefile.am
@@ -12,7 +12,7 @@ libgstcamerasrc_@GST_MAJORMINOR@include_HEADERS = \
 	gstcamerabin-enum.h \
 	gstbasecamerasrc.h
 
-libgstcamerasrc_@GST_MAJORMINOR@_la_CFLAGS = $(GST_PLUGINS_BASE_CFLAGS) $(GST_CFLAGS) -DGST_USE_UNSTABLE_API
+libgstcamerasrc_@GST_MAJORMINOR@_la_CFLAGS = $(GST_PLUGINS_BASE_CFLAGS) $(GST_CFLAGS) -DGST_USE_UNSTABLE_API -I$(top_srcdir)/gst-libs -I$(top_builddir)/gst-libs
 libgstcamerasrc_@GST_MAJORMINOR@_la_LIBADD = \
 	$(top_builddir)/gst-libs/gst/interfaces/libgstphotography-$(GST_MAJORMINOR).la \
 	$(GST_LIBS) $(GST_BASE_LIBS) $(GST_PLUGINS_BASE_LIBS) \
-- 
1.7.1

